plugins/org.polymap.kaps/src/org/polymap/kaps/ui/form/VertragsdatenBaulandSonstigesFormEditorPage.java

```diff
@@ -101,6 +98,32 @@ public void createFormContent( final IFormEditorPageSite site ) {
                 true );
         createFlaecheField( "Grundstücksbreite (m²)", vb.grundstuecksBreite(), right().top( lastLine ), client, true );
 
+        
+        ErtragswertverfahrenComposite ertragswertverfahren = ErtragswertverfahrenComposite.Mixin.forVertrag(vb.vertrag().get());
+        Double gesamtFlaeche = null;
+        if(ertragswertverfahren != null) {
+        	gesamtFlaeche = sum(
+        			new Object [] { ertragswertverfahren.flaecheZeile1(), ertragswertverfahren.wohnflaecheZeile1()},
+        			new Object [] { ertragswertverfahren.flaecheZeile2(), ertragswertverfahren.wohnflaecheZeile2()},
+        			new Object [] { ertragswertverfahren.flaecheZeile3(), ertragswertverfahren.wohnflaecheZeile3()},
+        			new Object [] { ertragswertverfahren.flaecheZeile4(), ertragswertverfahren.wohnflaecheZeile4()},
+        			new Object [] { ertragswertverfahren.flaecheZeile5(), ertragswertverfahren.wohnflaecheZeile5()},
+        			new Object [] { ertragswertverfahren.flaecheZeile6(), ertragswertverfahren.wohnflaecheZeile6()},
+        			new Object [] { ertragswertverfahren.flaecheZeile7(), ertragswertverfahren.wohnflaecheZeile7()},
+        			new Object [] { ertragswertverfahren.flaecheZeile8(), ertragswertverfahren.wohnflaecheZeile8()},
+        			new Object [] { ertragswertverfahren.flaecheZeile9(), ertragswertverfahren.wohnflaecheZeile9()},
+        			new Object [] { ertragswertverfahren.flaecheZeile10(), ertragswertverfahren.wohnflaecheZeile10()},
+        			new Object [] { ertragswertverfahren.flaecheZeile11(), ertragswertverfahren.wohnflaecheZeile11()},
+        			new Object [] { ertragswertverfahren.flaecheZeile12(), ertragswertverfahren.wohnflaecheZeile12()},
+        			new Object [] { ertragswertverfahren.flaecheZeile13(), ertragswertverfahren.wohnflaecheZeile13()},
+        			new Object [] { ertragswertverfahren.flaecheZeile14(), ertragswertverfahren.wohnflaecheZeile14()},
+        			new Object [] { ertragswertverfahren.flaecheZeile15(), ertragswertverfahren.wohnflaecheZeile15()}
+        	);
+        }
+        if(gesamtFlaeche != null) {
+        	vb.wohnflaeche().set(gesamtFlaeche);
```

Comment:
Ziehe diese Berechnung bitte mal in eine separate Methode, die du nach createFormContent aufrufst. Den Wert setze bitte nur im Form mit site.setFieldValue("wohnfläche") oderso ähnlich.